retriever/lib/engine_tools.py

```diff
@@ -162,6 +164,25 @@ def xml2csv(input_file, outputfile=None, header_values=None, row_tag="row"):
     return outputfile
 
 
+def xlsxcsv(book):
+    workbook = xlrd.open_workbook(book)
+    dire = book.rstrip(".xlsx")
+    if not os.path.exists(dire):
+        os.makedirs(dire)
+    else:
+        pass
+    os.chdir(dire)
+    res = len(workbook.sheet_names())
+    for sheet in range(0, res):
+        worksheet = workbook.sheet_by_index(sheet)
+        df = pd.DataFrame(columns=list(worksheet.row_values(0)))
+        for index in range(worksheet.nrows):
+            df.loc[len(df)] = worksheet.row_values(1)
+        table_name = workbook.sheet_names()
+        df.to_csv(table_name[sheet] + '.csv', index_label='index')
+    os.chdir("..")
```

Comment:
Is there a cleaner way to do this? I am not sure if this will always be compatible across different OS - it maybe. If we can pass the absolute path in `os.chdir()` somehow that would be great.